vars/gangplank.groovy

```diff
@@ -46,19 +46,28 @@ def generateSpec(params = [:]) {
     } else {
         gangplankCmd = "gangplank generate "
     }
-    gangplankCmd = getArtifacts(params, gangplankCmd)
-    gangplankCmd = getFlags(params, gangplankCmd)
+    gangplankCmd = _getArtifacts(params, gangplankCmd)
+    gangplankCmd = _getFlags(params, gangplankCmd)
     fileName = "/tmp/jobspec-${UUID.randomUUID()}.spec"
     gangplankCmd += "--yaml-out ${fileName} "
     runGangplank(gangplankCmd)
     return fileName
 }
-
+// Available parameters:
+//     configFile:   string  -- Name of the Minio config file previously created
+//     gangplankCmd  string  -- Gangplank command to run
 def startParallel(configFile, gangplankCmd) {
     gangplankCmd += "-m ${configFile}"
     runGangplank(gangplankCmd)
 }
 
+// Available parameters:
+//     artifacts:    list    -- list of artifacts to be built
+//     extraFlags    string  -- Extra flags to use
+//     image:        string  -- Name of the image to be used
+//     mode:         string  -- Gangplank Mode
+//     minioServeDir string  -- location to service minio from
+//     workDir:      string  -- cosa working directory
```

Comment:
Super minor: looks like we're inconsistently using `:` and also whether to capitalize the first word of the description. Pretty sure that's throughout the codebase though so nothing new. :)